src/R/Wpf/Impl/Microsoft.R.Wpf.csproj

```diff
@@ -36,6 +36,7 @@
     <Compile Include="..\..\..\GlobalAssemblyInfo.cs">
       <Link>Properties\GlobalAssemblyInfo.cs</Link>
     </Compile>
+    <Compile Include="BindableBase.cs" />
```

Comment:
Looks like this will be the third one, cause we already have one in Package and one in `Microsoft.Common.Wpf`